test/libsolidity/semanticTests/getters/transient_value_types_multi_frame_call.sol

```diff
@@ -15,7 +15,6 @@ contract C {
 }
 // ====
 // EVMVersion: >=cancun
-// compileViaYul: false
```

Comment:
Another set of test cases that came up in the docs PR:

```solidity
contract B {
    uint256 transient public value;

    function setValue(uint256 v) public {
        value += v;
    }
}

contract A {
    uint256 transient public value;

    function delegateSetValue(address otherContract, uint256 v) public {
        (bool success, ) = otherContract.delegatecall(abi.encodeWithSignature("setValue(uint256)", v));
        require(success);
    }

    function callSetValue(address otherContract, uint256 v) public {
        (bool success, ) = otherContract.call(abi.encodeWithSignature("setValue(uint256)", v));
        require(success);
    }
    function staticSetValue(address otherContract, uint256 v) view public returns (bool) {
        (bool success, ) = otherContract.staticcall(abi.encodeWithSignature("setValue(uint256)", v));
        return success;
    }
}

contract Test {
    A a = new A();
    B b = new B();

    function testDelegate() public returns (uint256, uint256) {
        a.delegateSetValue(address(b), 7);
        return (a.value(), b.value());
    }
    function testCall() public returns (uint256, uint256) {
        a.callSetValue(address(b), 8);
        return (a.value(), b.value());
    }
    function testStatic() view public returns (bool) {
        return a.staticSetValue(address(b), 0);
    }
}
```

Reply:
Another thing worth testing might be passing of transient storage variables into modifiers:
```solidity
contract C {
    address transient a;
    uint16 transient x;

    modifier m(uint16) {
        x += 10;
        _;
    }
    function f() public m(x) {
        x *= 10;
    }
}
```

It most likely works and goes through already implemented paths, but I'd add a test for it just in case.